jupyter-lite.json

```diff
@@ -0,0 +1,11 @@
+{
+  "jupyter-lite-schema-version": 0,
+  "jupyter-config-data": {
+    "disabledExtensions": [
+      "jupyterlab-kernel-spy",
+      "jupyterlab-tour",
```

Comment:
Do we need to disabled these extensions? It looks like they are not installed (not listed in the `requirements.txt`).